core/src/main/java/jenkins/management/EditDescriptionLink.java

```diff
@@ -0,0 +1,28 @@
+package jenkins.management;
+
+import hudson.Extension;
+import hudson.model.ManagementLink;
+import org.jenkinsci.Symbol;
+
+@Extension(ordinal = Integer.MAX_VALUE - 800) @Symbol("editDescription")
```

Comment:
Does the ordinal refers to the position of the element on the page? If so, what is the recommended value?